source/kraken/configuration.cpp

```diff
@@ -29,162 +29,172 @@ www.navitia.io
-        ("GENERAL.is_realtime_add_enabled", po::value<bool>()->default_value(false),
-                                        "enable loading of realtime data that add stop_times or trips")
-        ("GENERAL.kirin_timeout", po::value<int>()->default_value(60000),
-                                  "timeout in ms for loading realtime data from kirin")
-        ("GENERAL.kirin_retry_timeout", po::value<int>()->default_value(5*60*1000),
-                                  "timeout in ms before retrying to load realtime data")
-        ("GENERAL.slow_request_duration", po::value<int>()->default_value(1000),
-                                  "request running a least this number of milliseconds are logged")
-
-        ("GENERAL.display_contributors", display_contributors ?
-             po::value<bool>()->default_value(*display_contributors) : po::value<bool>()->default_value(false),
-         "display all contributors in feed publishers")
-        ("GENERAL.raptor_cache_size", po::value<int>()->default_value(10), "maximum number of stored raptor caches")
-        ("GENERAL.log_level", po::value<std::string>(), "log level of kraken")
-        ("GENERAL.log_format", po::value<std::string>()->default_value("[%D{%y-%m-%d %H:%M:%S,%q}] [%p] [%x] - %m %b:%L  %n"), "log format")
-
-        ("GENERAL.metrics_binding", po::value<std::string>(), "IP:PORT to serving metrics in http")
-
-        ("BROKER.host", po::value<std::string>()->default_value("localhost"), "host of rabbitmq")
-        ("BROKER.port", po::value<int>()->default_value(5672), "port of rabbitmq")
-        ("BROKER.username", po::value<std::string>()->default_value("guest"), "username for rabbitmq")
-        ("BROKER.password", po::value<std::string>()->default_value("guest"), "password for rabbitmq")
-        ("BROKER.vhost", po::value<std::string>()->default_value("/"), "vhost for rabbitmq")
-        ("BROKER.exchange", po::value<std::string>()->default_value("navitia"), "exchange used in rabbitmq")
-        ("BROKER.rt_topics", po::value<std::vector<std::string>>(), "list of realtime topic for this instance")
-        ("BROKER.timeout", po::value<int>()->default_value(100), "timeout for maintenance worker in millisecond")
-        ("BROKER.sleeptime", po::value<int>()->default_value(1), "sleeptime for maintenance worker in second")
-
-        ("CHAOS.database", po::value<std::string>(), "Chaos database connection string");
+         "path to the zmq socket used for receiving resquest")(
+            "GENERAL.instance_name",
+            name ? po::value<std::string>()->default_value(*name) : po::value<std::string>()->required(),
+            "name of the instance")
+
+            ("GENERAL.nb_threads", po::value<int>()->default_value(1), "number of workers threads")(
+                "GENERAL.is_realtime_enabled", po::value<bool>()->default_value(false),
+                "enable loading of realtime data")("GENERAL.is_realtime_add_enabled",
```

Comment:
TBH, it was already ugly :rofl: 
You can disable the linting with : https://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code.
Although, this might require some tweaks in the config if we don't want to go back to all the unit tests that are doing things in a similar way.